src/TodoListItem.js

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 
-function TodoListItem(props) {
-  return <li key={props.todo.id}>{props.todo.title}</li>
+function TodoListItem(TodoListItem) {
```

Comment:
same problem here as in AddTodo Form component 
you refer to props incorrectly 
Todo ListItem(props) was the correct way to accept props please, revise